EduNLP/Vector/t2v.py

```diff
@@ -52,6 +60,7 @@ def get_pretrained_t2v(name, model_dir=MODEL_DIR):
         )
     url, model_name, *args = PRETRAINED_MODELS[name]
     model_path = get_data(url, model_dir)
-    if model_name in ["d2v"]:
-        model_path = path_append(model_path, os.path.basename(model_path) + ".bin", to_str=True)
+    if model_name in ["d2v", "w2v"]:
+        postfix = ".bin" if model_name == "d2v" else ".kv"
```

Comment:
why kv in W2V

Reply:
The w2v model is trained as .kv as default instead of .bin, containing only the word2vec dictonary and more smaller.

Reply:
Got it